MANIFEST

```diff
@@ -0,0 +1,9 @@
+Changes
+lib/Pg/Reindex.pm
```

Comment:
cool. You converted the script to a module.

Now it would be nice to add a script that uses that module.